modules/consensus/synchronize.go

```diff
@@ -176,6 +176,7 @@ func (cs *ConsensusSet) sendBlocks(conn modules.PeerConn) error {
 			found = true
 			// Start from the child of the common block.
 			start = pb.Height + 1
+			break
```

Comment:
Let's assume an average block size of 100kB, with an average network size of 100 peers. There are ~40,000 blocks. Let's also assume constant block size and number of peers.

`100kB * 100 peers * sum of integers from 0 to 40,000 ~= 8PB`

Wow. Did I get my math wrong?